docs/azure/sdk/includes/implement-managed-identity-concepts.md

```diff
@@ -0,0 +1,13 @@
+---
+ms.topic: include
+ms.date: 02/12/2025
+---
+
+## Authenticate to Azure services from your app
+
+The [Azure.Identity](/dotnet/api/azure.identity) library provides various implementations of `TokenCredential` for different scenarios and credential types. To authenticate to Azure services from your app code using Microsoft Entra ID and a managed identity, use the [`ManagedIdentityCredential`](/dotnet/api/azure.identity.managedidentitycredential?view=azure-dotnet) class. These classes allow apps to seamlessly authenticate to Azure resources whether the app is running locally, deployed to Azure, or on an on-premises server.
+
+The steps ahead demonstrate how to use a `TokenCredential` across two different environments:
```

Comment:
```suggestion
The [Azure Identity library](/dotnet/api/azure.identity?view=azure-dotnet&preserve-view=true) provides various *credentials*&mdash;implementations of `TokenCredential` adapted to supporting different scenarios and Microsoft Entra authentication flows. Since managed identity is unavailable when running locally, the steps ahead demonstrate which credential to use in which scenario:
```